lib/Remote/RemoteWebDriver.php

```diff
@@ -149,15 +182,17 @@ public static function createBySessionID(
         $connection_timeout_in_ms = null,
         $request_timeout_in_ms = null
     ) {
-        $executor = new HttpCommandExecutor($selenium_server_url);
+        // BC layer to not break the method signature
```

Comment:
should you trigger a deprecation warning if a child class extends this method without adding the argument ?

Reply:
I've not done added it because the library currently doesn't trigger any deprecation. But why not. WDYT @OndraM?